pycqed/instrument_drivers/physical_instruments/DDM/DDMq.py

```diff
@@ -0,0 +1,318 @@
+'''
```

Comment:
What is the difference between the DDM module and the DDMq? 

This is quite unclear to me, which one do I need to review?